dev-python/e4u/e4u-0.1_rc4.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 eapi7-ver
+
+MY_PV="$(ver_rs 2 '' )"
+
+DESCRIPTION="It's the bundle library for emoji4unicode"
+HOMEPAGE="https://github.com/lambdalisue/e4u"
+SRC_URI="https://github.com/lambdalisue/e4u/archive/0.1rc4.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-python/beautifulsoup:python-2[${PYTHON_USEDEP}]"
+
+DEPEND="${DEPEND}"
```

Comment:
Missing dependency of setuptools. Also check if it uses entry_points, in which setuptoots would be RDEP too.

Reply:
I am not sure, if I understand it, but I tend to say yes after looking.